lib/if.c

```diff
@@ -1092,6 +1093,40 @@ const char *if_link_type_str(enum zebra_link_type llt)
 	return NULL;
 }
 
+bool if_link_params_cmp(struct if_link_params *iflp1,
+			struct if_link_params *iflp2)
+{
+	struct if_link_params iflp1_copy, iflp2_copy;
+
+	memcpy(&iflp1_copy, iflp1, sizeof(struct if_link_params));
+	memset(&iflp1_copy.ext_admin_grp, 0, sizeof(struct admin_group));
+
+	memcpy(&iflp2_copy, iflp2, sizeof(struct if_link_params));
+	memset(&iflp2_copy.ext_admin_grp, 0, sizeof(struct admin_group));
```

Comment:
What is the goal of these lines? iflp1_copy and iflp2_copy are not used later for the comparison

Reply:
fixed